docs/IntegrationGuide.md

```diff
@@ -20,8 +21,50 @@ This documentation describes how to setup, start and use `commercetools-paypal-p
 
 ## Preparing commercetools Platform accounts
 
+  - create *`payment-paypal`* custom type with fields, specified in 
+  [ctPaymentCustomType.json](/src/main/resources/referenceModels/ctPaymentCustomType.json). See also 
```

Comment:
Maybe this step can be automated, just create the type/field automatically, when they doesn`t exist

Reply:
#40